lib/js/tt.js

```diff
@@ -709,9 +731,9 @@ window.tsfTT = function () {
 		if ( element?.[0] )
 			element = element[0];
 
-		return element?.classList.contains( ttNames.base )
-			? element
-			: element?.querySelector( ttSelectors.base );
+		return ttMap.get( element ) || ttMap.get(
+			element?.querySelector( ttSelectors.base )
+		);
```

Comment:
If I'm not mistaken I just realized this change isn’t BC. The existing logic returns `element` if it **is** a tooltip. The changed logic tries to lookup a tooltip by the `element`, so if `element` is a tooltip itself, then it’d return `undefined`.